dnf/cli/cli.py

```diff
@@ -214,13 +214,12 @@ def do_transaction(self, display=()):
             elif 'test' in self.conf.tsflags:
                 logger.info(_("{prog} will only download packages, install gpg keys, and check the "
                               "transaction.").format(prog=dnf.util.MAIN_PROG_UPPER))
-            if dnf.util.is_container():
-                _container_msg = _("""
-*** This system is managed with ostree.  Changes to the system
-*** made with dnf will be lost with the next ostree-based update.
-*** If you do not want to lose these changes, use 'rpm-ostree'.
+            if dnf.util.is_bootc_host():
+                _bootc_host_msg = _("""
+*** error: system is configured to be read-only; for more
```

Comment:
Could you start a sentence with an upper-case letter? I.e. spell "error:" as "Error:".

Reply:
fixed